pss/pss.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
update copyright, year and authors are wrong

Reply:
fixed in all files